games/main.tex

```diff
@@ -134,46 +123,38 @@ \section{Introduction}
+An alternative strategy to prevent double cashing, when repeated payments are made to the same beneficiary, the cheques contain the \emph{cumulative} total amount ever credited to the beneficiary. The total cumulative amount that has been cashed out is stored in the contract for each beneficiary. When a new cheque is submitted, the contract ignores cheques with amount equal or less than the stored total, but it will transfer the difference if it receives a cheque with a higher
 total.
-This simple trick also makes it possible to cash cheques in bulk by sending only the last one in
-thereby further reducing transaction costs. Incidentally, the cumulative amount
-directly represents the total of all outgoing payments and serves as a credit history for the owner.
-
-The amount deposited in the chequebook (\gloss{global balance})
-serves as collateral pooled over the beneficiaries of all outstanding cheques.
-In this simplest form the chequebook has the same liability as real-world cheques: none.
-Since funds can be freely moved out of the chequebook wallet, solvency cannot be guaranteed
-at the time of cashing.
-
-Parts of this deposit can be locked (\gloss{global deposit}).
-If the frequency of issuing and the amount promised are limited,
-the global balance can be informational about the likelihood of insolvency.
-If, however, there is no restrictions on issuing and paying out cheques,
-the owner can issue cheques totalling more than what is backed
-by the chequebook's global balance. Locking, by itself, provides no further guarantees, so accepting
-cheques constitutes a risk to the beneficiary.
+
+This simple trick also makes it possible to cash cheques in bulk, because only the current `last cheque' must ever be processed. This achieves the reduction of transaction costs alluded to above.
+
+Incidentally, the cumulative amount stored in the contract, represents the total of all outgoing payments that have been honoured and thus it serves as a credit history for the owner.
+
+The amount deposited in the chequebook (\gloss{global balance}) serves as collateral for the cheques. It is pooled over the beneficiaries of all outstanding cheques.
+In this simplest form the chequebook has the same guarantee as real-world cheques: none. When a chequebook contract has zero balance, and cheque that is submitted will bounce. Since funds can be freely moved out of the chequebook wallet at any time, solvency at the time of cashing can never be guaranteed. This is the trade off between transaction costs and risk of settlement failure.
+
+% Parts of this deposit can be locked (\gloss{global deposit}).
```

Comment:
we skip this?

Reply:
unless you can explain it to me and say why it should stay there... For now I say we can do without it.